R/rABS.R

```diff
@@ -0,0 +1,94 @@
+#' ABS simulation
+#'
+#' This function is for simulating the Auto-correlated Bayesian Sampler \insertCite{zhuAutocorrelatedBayesian2023}{samplr}.
+#'
+#' @param dec_bdry The decision boundary that separates the posterior hypothesis distribution.
+#' @param discrim The stimuli discriminability.
+#' @param delta The threshold of the relative difference. It should be an integer.
+#' @param nd_time The non-decision time.
+#' @param s_nd_time The range of the non-decision time.
+#' @param lambda The rate parameter of the gamma distribution for decision time.
+#' @param trial_fdbk The feedback of each trial. It should be a character only consist of 0 (below the decision boundary) and/or 1 (above the decision boundary).
```

Comment:
I don't think it should be a character but a numeric vector? Please correct me if I'm wrong